Makefile

```diff
@@ -12,7 +12,7 @@ backend:
 	-cd backend && npm run dev
 
 database:
-	-docker-compose up -d postgres pgadmin
+	-docker-compose up -d postgres
```

Comment:
pgadmin has been commented in docker-compose file but we don't need it for development

Reply:
Sorry, forget to remove pgadmin in Makefile. Have not tested enough...